geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -115,14 +115,28 @@
                             <h2>{{ image_info.shelfmark }} {{ image_info.label }}</h2>
                             {% if not image_info.excluded %}
                                 {% if edit_mode %}
+                                    {# popout and close button #}
                                     <button class="primary popout-close-button" type="button" aria-label="Close"><i class="ph-x"></i></button>
                                     <button class="popout-button" type="button">
                                         <svg>
                                             <use xlink:href="{% static 'img/ui/desktop/all/pop-out.svg' %}#pop-out-icon" />
                                         </svg>
                                         <span class="sr-only">Pop out</span>
                                     </button>
+                                    <fieldset class="toolbar">
+                                        {# rectangle tool #}
+                                        <label class="rect-tool active-tool">
+                                            <input type="radio" checked />
+                                            <span class="sr-only">Rectangle tool</span>
```

Comment:
not sure how meaningful "rectangle tool" is; how about rephrasing to describe better what these do?

```suggestion
                                            <span class="sr-only">Select a rectangle</span>
```

Reply:
Will put this suggestion into the tahqiq update, thanks!